contracts/integrations/test/exchange/fill_dydx_order_test.ts

```diff
@@ -0,0 +1,232 @@
+import { DydxBridgeActionType, dydxBridgeDataEncoder, TestDydxBridgeContract } from '@0x/contracts-asset-proxy';
+import { DummyERC20TokenContract } from '@0x/contracts-erc20';
+import { blockchainTests, constants, describe, expect, toBaseUnitAmount } from '@0x/contracts-test-utils';
+import { SignedOrder } from '@0x/order-utils';
+import { BigNumber } from '@0x/utils';
+import { DecodedLogArgs, LogWithDecodedArgs } from 'ethereum-types';
+import * as _ from 'lodash';
+
+import { deployDydxBridgeAsync } from '../bridges/deploy_dydx_bridge';
+import { Actor } from '../framework/actors/base';
+import { Maker } from '../framework/actors/maker';
+import { Taker } from '../framework/actors/taker';
+import { DeploymentManager } from '../framework/deployment_manager';
+
+blockchainTests.resets('Exchange fills dydx orders', env => {
+    let testContract: TestDydxBridgeContract;
+    let makerToken: DummyERC20TokenContract;
+    let takerToken: DummyERC20TokenContract;
+    const marketId = new BigNumber(3);
+    const dydxConversionRateNumerator = toBaseUnitAmount(6);
+    const dydxConversionRateDenominator = toBaseUnitAmount(1);
+    let maker: Maker;
+    let taker: Taker;
+    let orderConfig: Partial<SignedOrder>;
+    let dydxBridgeProxyAssetData: string;
+    const defaultDepositAction = {
+        actionType: DydxBridgeActionType.Deposit as number,
+        accountId: constants.ZERO_AMOUNT,
+        marketId,
+        conversionRateNumerator: dydxConversionRateNumerator,
```

Comment:
Can we also test the behavior when the conversion rate would trigger a rounding error?